provision/rosa-cross-dc/keycloak-benchmark-crossdc-tests/src/test/java/org/keycloak/benchmark/crossdc/client/KeycloakClient.java

```diff
@@ -137,7 +139,10 @@ public Map<String, Object> refreshToken(String realmName, String refreshToken, S
     }
 
     public String usernamePasswordLogin(String realmName, String username, String password, String clientId) throws IOException, URISyntaxException, InterruptedException {
-        String formUrl = openLoginForm(realmName, clientId);
+        HttpResponse<String> loginFormResponse = openLoginForm(realmName, clientId);
+        assertEquals(200, loginFormResponse.statusCode(), "Failed to open login form for realm " + realmName + " with response body " + loginFormResponse.body());
+        String formUrl = getLoginFormActionURL(loginFormResponse);
```

Comment:
we are returning `null` in one of the cases for `getLoginFormActionURL`, should we handle that when we create this `formUrl` variable ?